pkg/apis/kubeone/types.go

```diff
@@ -322,6 +322,9 @@ type CloudProviderSpec struct {
 	// External
 	External bool `json:"external,omitempty"`
 
+	// DisableBundledCSIDriver used to cancel automatic deploy of bundled CSI drivers
```

Comment:
Nit: potential improvements
```suggestion
	// DisableBundledCSIDriver disables automatic deployment of CSI drivers bundled with KubeOne
```

Reply:
done